translations/daterangepicker.fr.xliff

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="fr" datatype="plaintext" original="daterangepicker.en.xliff">
+        <body>
+            <trans-unit id="daterangepicker.today">
+                <source>daterangepicker.today</source>
+                <target>Aujourd'hui</target>
+            </trans-unit>
+            <trans-unit id="daterangepicker.yesterday">
+                <source>daterangepicker.yesterday</source>
+                <target>Hier</target>
+            </trans-unit>
+            <trans-unit id="daterangepicker.lastWeek">
+                <source>daterangepicker.lastWeek</source>
+                <target>Semaine dernière</target>
+            </trans-unit>
+            <trans-unit id="daterangepicker.thisWeek">
+                <source>daterangepicker.thisWeek</source>
+                <target>Cette semaine</target>
+            </trans-unit>
+            <trans-unit id="daterangepicker.lastMonth">
+                <source>daterangepicker.lastMonth</source>
+                <target>Mois dernier</target>
+            </trans-unit>
+            <trans-unit id="daterangepicker.thisMonth">
+                <source>daterangepicker.thisMonth</source>
+                <target>Ce mois</target>
+            </trans-unit>
+            <trans-unit id="daterangepicker.lastYear">
+                <source>daterangepicker.lastYear</source>
+                <target>L'an dernier</target>
+            </trans-unit>
+            <trans-unit id="daterangepicker.thisYear">
+                <source>daterangepicker.thisYear</source>
+                <target>Cette année</target>
+            </trans-unit>
+            <trans-unit id="daterangepicker.customRange">
```

Comment:
This needs to be merged, its duplicated (see below)

Reply:
Sorry! I just deleted the duplicate ID.